apisix/plugins/ext-plugin/init.lua

```diff
@@ -273,6 +274,12 @@ local function handle_extra_info(ctx, input)
 
         local var_name = var_req:Name()
         res = ctx.var[var_name]
+    elseif info_type == extra_info.ReqBody then
+        local info = req:Info()
+        local reqbody_req = extra_info_reqbody.New()
+        reqbody_req:Init(info.bytes, info.pos)
+
+        res = core.request.get_body()
```

Comment:
Better to log down err if it is not nil.

Reply:
done